modules/asset/ui/src/index.js

```diff
@@ -1,16 +1,5 @@
-export default () => {
-  const localesSelectorElem = document.querySelector('[data-locales-selector]');
-  const toggleBtn = localesSelectorElem.querySelector('[data-locales-toggle]');
-  const localesList = localesSelectorElem.querySelector('.locales__list');
-
-  window.addEventListener('click', ({ target }) => {
-    if (localesList.classList.contains('active') && !localesSelectorElem.contains(target)) {
-      localesList.classList.remove('active');
-    }
-  });
+import localesSelector from './_locales';
```

Comment:
You don't have to `_` prefix things because only `index.js` is loaded automatically.